esp-hal/src/rtc_cntl/mod.rs

```diff
@@ -788,7 +788,10 @@ impl Rwdt {
         let rtc_cntl = unsafe { &*LP_WDT::PTR };
 
         self.set_write_protection(false);
-        rtc_cntl.wdtfeed().write(|w| unsafe { w.bits(1) });
+        #[cfg(any(esp32c6, esp32h2))]
```

Comment:
This sucks, but I already published new PACs for the upcoming release. I will update the PACs in the repo, then make a note to fix this after we've published.